cmd/kubeadm/app/cmd/phases/init/certs_test.go

```diff
@@ -41,6 +41,12 @@ func (t *testCertsData) CertificateDir() string             { return t.cfg.Certi
 func (t *testCertsData) CertificateWriteDir() string        { return t.cfg.CertificatesDir }
 
 func TestCertsWithCSRs(t *testing.T) {
+	// restore global variables
+	defer func() {
```

Comment:
Jordan showed me the way :)
https://github.com/kubernetes/kubernetes/pull/100203#discussion_r593563809

Reply:
I mean… not using global vars anymore is better… but if the thing you're testing explicitly requires them, resetting once the test is over is good to do